tests/about.v.out

```diff
@@ -100,6 +100,12 @@ HB: AddAG.sort is a canonical projection
 HB: AddAG.sort has the following canonical values:
     - Z 
     - Ring.sort (from "./examples/demo1/hierarchy_5.v", line 200)
+    - Ring_of_TYPE.sort (from "./examples/demo1/hierarchy_5.v", line 171)
+    - Ring_of_AddComoid.sort (from "./examples/demo1/hierarchy_5.v", line 148)
+    - Ring_of_AddAG.sort (from "./examples/demo1/hierarchy_5.v", line 114)
+    - AddAG_of_TYPE.sort (from "./examples/demo1/hierarchy_5.v", line 51)
+    - AddAG_of_AddComoid.sort (from "./examples/demo1/hierarchy_5.v", line 44)
+    - @eta 
```

Comment:
I think these should be filtered out one way or another.

Reply:
I still think it is nice to list them, but maybe in another category...

Reply:
In another PR though